test/test-applications/integrations/TestApplication.AspNet.NetFramework/Dockerfile

```diff
@@ -4,6 +4,11 @@ ARG windowscontainer_version=ltsc2022
 FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-${windowscontainer_version}
 ARG configuration=Debug
 ARG platform=x64
+WORKDIR C:\opentelemetry
+COPY bin/tracer-home .
+WORKDIR C:\init
+COPY docker-init.ps1 .
+RUN [ "powershell", "-c", "C:\\init\\docker-init.ps1" ]
```

Comment:
Cannot we simply do something like the following:

1. copy the `OpenTelemetry.DotNet.Auto.psm1`
2. set `OTEL_DOTNET_AUTO_INSTALL_DIR` env var to `C:\opentelemetry`
3. call in powershell `Import-Module OpenTelemetry.DotNet.Auto.psm1; Register-OpenTelemetryForIIS`

instead of having `docker-init.ps1`?

PS. It would be cool if  `Install-OpenTelemetryCore` could also operate on extracted files (not only zip archive)

Reply:
#2575 created